email.go

```diff
@@ -33,6 +34,14 @@ func (r *EmailClient) Send(to []string, mimeType, subject, body string) error {
 	m.SetHeader("To", to[0])
 	m.SetHeader("Subject", subject)
 	m.SetBody(mimeType, body)
+	if mimeType == HTML_MIME_TYPE {
+		plaintext, err := html2text.FromString(body)
+		if err != nil {
+			logcabin.Info.Println(err)
+		} else {
+			m.AddAlternative("text/plain", plaintext)
```

Comment:
gomail seems to suggest it's better to do the first SetBody with the `text/plain` version, but it didn't seem to matter at least in my mail client. We can change it if someone's bothered, probably.

Reply:
I just got an analysis status change email in QA, and in GMail in my Firefox browser, it's displaying the plain-text version instead of the HTML version. Other emails I have received that display as HTML are putting the plain-text version in the email body before the HTML version.

Reply:
Ah, okay, so we should probably swap this around, I guess. gmail is pretty widely used. I can make that change and push it out; @psarando would you be on board to generate a test email to yourself afterwards to make sure it appears nicely in gmail?